pkg/reconciler/autoscaling/resources/target_test.go

```diff
@@ -137,9 +164,13 @@ func TestResolveMetricTarget(t *testing.T) {
 			if tc.cfgOpt != nil {
 				cfg = tc.cfgOpt(*cfg)
 			}
-			if gotTgt, _ := ResolveMetricTarget(tc.pa, cfg); gotTgt != tc.wantTgt {
+			gotTgt, gotTot := ResolveMetricTarget(tc.pa, cfg)
+			if gotTgt != tc.wantTgt {
 				t.Errorf("ResolveMetricTarget(%v, %v) = %v, want %v", tc.pa, config, gotTgt, tc.wantTgt)
 			}
+			if gotTot != tc.wantTot {
+				t.Errorf("ResolveMetricTarget(%v, %v) = %v, want %v", tc.pa, config, gotTot, tc.wantTot)
```

Comment:
We should unify these to one statement, i.e.

```golang
if gotTgt != tc.wantTgt || gotTot != tc.wantTot {
  t.Errorf("ResolveMetricTarget(%v, %v) = (%v, %v) want (%v, %v)", tc.pa, config, gotTgt, gotTot, tc.wantTgt, tc.wantTot)
}
```

Error messages will be quite confusing otherwise I fear.

Reply:
Done.